assets/i18n/fr.json

```diff
@@ -54,7 +52,8 @@
     "timerIsUp": {
         "announce":  "Votre minuteur est terminé.",
         "announce_name":  "Votre minuteur {{ name }} est terminé.",
-        "addTime": "Ok, je rajoute {{ time }} à votre minuteur."
+        "addTime": "Ok, je rajoute {{ time }} à votre minuteur.",
+        "addTime_name": "Ok, je rajoute {{ time }} à votre minuteur{{ name }}."
```

Comment:
manque un espace @thomas-bouvier  @Rockingmai

Reply:
Fixed in 3461fcfc7aa2e31735ce1a37d566c0676ac1f86f